### ISSUE TITLE
Accessibility Standards

### ISSUE THREAD

**Issue by ThienDW** [shanebrownescope/engr-science-teaching-resources#137]
**User Story Statement:**
As an admin of the web application, I want to ensure that my system complies with accessibility standards so that I know my users will be able to interact with the site.

**Background:**
Maintaining accessibility standards are vital for any digital interface, but it is especially essential for educational materials, where special attention must be paid so that persons with disabilities are not put at any disadvantage. For these purposes, the WCAG guidelines are the most straightforward rules to follow to ensure universal design. 

**Solution/Requirements**

**Functional:**
The system shall comply with WCAG's accessibility guidelines.

**Nonfunctional:**
Users should have an easy, intuitive experience navigating the system.

**Design Resources**
Web Content Accessibility Guidelines 2.1 - https://www.w3.org/TR/WCAG21/

**Dependencies**
N/A

**Tasks/Subtasks**
Research specific standards needed
Begin evaluating individual standards
- Variety of tests may be needed depending on which criteria is being evaluated

Identify specific failures in the system, and work with project team to create a plan for addressing each one.

Time Estimate

2 weeks

Definition of Done

All criteria have been evaluated
Guideline violations have been noted and a plan for addressing them has been created
